### ISSUE TITLE
Refactor for project clone

### ISSUE THREAD

**Issue by burnhamd** [joreilly/PeopleInSpace#29]
This is one of the best examples of getting started with a KMM project and the only example Ive seen that uses SwiftUI and jetpack compose. 

It would be great if there was a refactor guide on how to migrate to a new project/package names

**Comment by miduch** [comment]
any plans to update project to use SwiftUI 2 (from swiftUI app using SceneDelegate to the new @main App protocol)?

**Comment by joreilly** [comment]
Yeah,  definitely something I'd hoped to do.  Also open to PRs :)

**Comment by joreilly** [comment]
@burnhamd sorry, meant to respond to this before.  So, I did update settings for this (and other repos) the other day to be a "Template repository" but still needs more updates to allow as you migration to different project/package name.

**Comment by RdeWilde** [comment]
I'm also looking for a smooth way to do this.
Basically I'd like to start of multiple projects from the same base, and merge new update back into it.
Now sure what structure would be best. I was thinking to keep platform-specific boilerplate code, and base the project-specific implementation in a subproject of it? That way I hope I can use keep the base projects up to date, and not having to do it on a per-project base.

But I'm only just getting started, thus very interested in other perspectives

**Comment by RdeWilde** [comment]
> @burnhamd sorry, meant to respond to this before. So, I did update settings for this (and other repos) the other day to be a "Template repository" but still needs more updates to allow as you migration to different project/package name.

While using it as a template is a great idea, i experienced at least one caveat, which is that you can merge from and to another. 
```
fatal: refusing to merge unrelated histories
```
It seems to create a whole new repo.
